### ISSUE TITLE
Error with 1.3.1

### ISSUE THREAD

**Issue by smefa** [dsoares/roundcube-rcguard#38]
I got an error with Roundcube 1.4.9 and rcguard 1.3.1
syntax error, unexpected '=' in \plugins\rcguard\rcguard.php on line 427
So I changed from:
[$subnet, $bits] = explode('/', $cidr);
to
[$subnet, $bits] == explode('/', $cidr);

I have no idea if it is correct but I can login now  :)

**Comment by marki555** [comment]
It is not correct :) but affects only the subnet/CIDR calculation.
`explode()` function returns an array and this line tries to assign first two array elements directly to the mentioned variables. However this is supported only since PHP 7.1+ ([link](https://www.php.net/manual/de/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring)).
You can use this code instead:
```
        //[$subnet, $bits] = explode('/', $cidr);
        $A=explode('/', $cidr);
        $subnet=$A[0];
        $bits=$A[1];

```

**Comment by smefa** [comment]
It works thanks!

**Comment by marki555** [comment]
I don't think the issue should be closed as it still needs to be fixed in the repository by the rcguard developer (or correctly say in README that it requires PHP 7.1 and not 5.4).
Also keeping the issue opened will help other people find it more easily until it is fixed.

**Comment by dsoares** [comment]
It does not require PHP 7.1, it was just a mistake with that line when moving to the short array syntax. 
Fixed with [release 1.3.2](https://github.com/dsoares/roundcube-rcguard/releases/tag/1.3.2). Thank you all for the warning!
